backend/src/services/GenerateReporter/GenerateReportService.ts

```diff
@@ -70,6 +71,15 @@ export class GenerateReportService {
   async generateReporter(
     request: GenerateReportRequest
   ): Promise<GenerateReporterResponse> {
+
+    if (Object.keys(request).length === 0) {
```

Comment:
Object.keys(request).length === 0 
does it mean
request === {}

Reply:
Yes, it does that.

Reply:
> Yes, it does mean that.

so could you please make the code more easy to read

Reply:
OK, I will fix this issue soon.